src/content/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview.mdx

```diff
@@ -16,7 +16,7 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-If you haven't already, [sign up for a free New Relic account](https://newrelic.com/signup) so you can instrument your systems and send telemetry data to New Relic. Our guided install creates a customized CLI command for your environment that downloads and installs the New Relic CLI and the infrastructure agent.
+If you haven't already, [sign up for a free New Relic account](https://newrelic.com/signup) so you can instrument your systems and send telemetry data to New Relic. Our guided install creates a customized CLI command for your environment that downloads and installs the New Relic CLI and the infrastructure agent. Before you begin, it may be helpful to read [Best practices for infrastructure monitoring](/docs/new-relic-solutions/best-practices-guides/full-stack-observability/infrastructure-monitoring-best-practices-guide/).
```

Comment:
```suggestion
If you haven't already, [sign up for a free New Relic account](https://newrelic.com/signup) so you can instrument your systems and send telemetry data to New Relic. Our guided install creates a customized CLI command for your environment that downloads and installs the New Relic CLI and the infrastructure agent. Before you begin, it may be helpful to read [Infrastructure monitoring best practices guide](/docs/new-relic-solutions/best-practices-guides/full-stack-observability/infrastructure-monitoring-best-practices-guide/).
```